draft-bormann-cbor-yang-standin.md

```diff
@@ -492,13 +526,21 @@ They already are CBOR tags and thus in the registry, but might
 get lost in the bulk of that (and are only identified as YANG-CBOR
 stand-in Tags in the specification).
 
+I think we should create some registry saying "this tag may encode that types"
+and vice versa, the problem may be how to define this registry well. --Maria
+
 ## media-type parameters
 
-ISSUE: Should the use of stand-in tags be mentioned in the various
-YANG-CBOR-based media types (as a media type parameter)?
+IANA is requested to register a media-type subparameter `standin`
```

Comment:
This parameter does not make a lot of sense for ACCEPT.

Reply:
Not exactly sure what you're here about.